Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1419,21 +1406,14 @@ def search_domain_command(args):
         else:
             md = tableToMarkdown(table_name, autofocus_domain_output)
 
-        human_readable += md
-
-        domain_indicator_list.append(domain)
-        raw_response.append(raw_res)
-        autofocus_domain_list.append(autofocus_domain_output)
-
-    command_results = CommandResults(
-        outputs_prefix='AutoFocus.Domain',
-        outputs_key_field='IndicatorValue',
-        outputs=autofocus_domain_list,
-
-        readable_output=human_readable,
-        raw_response=raw_response,
-        indicators=domain_indicator_list
-    )
+        command_results.append(CommandResults(
+            outputs_prefix='AutoFocus.Domain',
+            outputs_key_field='IndicatorValue',
+            outputs=autofocus_domain_output,
+            readable_output=md,
+            raw_response=raw_res,
+            indicators=[domain]
```

Comment:
change this to be `indicator=domain`
leave the `indicators` arg in `__init__` too but use only `indicator`